src/data/markdown/docs/30 xk6-browser/01 Get started/03 Metrics.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Browser performance metrics'
```

Comment:
Alternative titles:
- Browser metrics
- Built-in metrics
- Browser built-in metrics
- Metrics